addons/hr_holidays/models/resource.py

```diff
@@ -157,6 +160,57 @@ def unlink(self):
 
         return res
 
+    @api.model
+    def load_public_holidays(self, companies=[], from_ui=False):
+        company_ids = self.env['res.company'].browse(companies)
+        if not company_ids:
+            return
```

Comment:
The early return should be before the browse + the variable names
```suggestion
        if not company_ids:
            return
        companies = self.env['res.company'].browse(company_ids)
```